Doppler.BillingUser/Controllers/BillingController.cs

```diff
@@ -242,7 +242,7 @@ public async Task<IActionResult> CreateAgreement([FromRoute] string accountname,
                     return new BadRequestObjectResult("Invalid selected plan");
                 }
 
-                if (newPlan.IdUserType != UserTypeEnum.INDIVIDUAL && newPlan.IdUserType != UserTypeEnum.MONTHLY)
+                if (newPlan.IdUserType != UserTypeEnum.INDIVIDUAL && newPlan.IdUserType != UserTypeEnum.MONTHLY && newPlan.IdUserType != UserTypeEnum.SUBSCRIBERS)
```

Comment:
Que opinás de, siguiendo [la regla de los tres](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)), hacer una lista en lugar de tantas condiciones? 

```suggestion
                if (!allowedPlanTypes.Includes(newPlan.IdUserType))
```

Reply:
muy buena idea, lo modifico.

Reply:
Debería definir una lista de enums?:
private readonly List<UserTypeEnum> allowedPlanTypes = new List<UserTypeEnum> { UserTypeEnum.INDIVIDUAL, UserTypeEnum.MONTHLY, UserTypeEnum.SUBSCRIBERS };

Reply:
Podría ser, o incluso un HashSet.

Tal vez lo puedas poner como `static readonly` si encontrás un nombre más específico.

Reply:
que te parece PlansTypesAllowedForBilling?

Reply:
No tengo opinión porque no estoy tan metido en contexto, pero suena bien. Solo dos detalles, creo que no va el plural en plan y creo que el adjetivo iría mejor adelante: `AllowedPlanTypesForBilling` o `BillingAllowedPlanTypes`